README.md

```diff
@@ -298,6 +298,12 @@ Here are the steps to resolve this issue:
    3. Paste the password into the **Google account app password** field.
    4. Click **Submit**.
 
+### Devices found, but not initialized
+
+If the debug logs list your devices, but then show `Successfully initialized 0 Google Home devices` make sure you are logged into the correct Google account.
+Your account has to be able to access the devices in your Home.
+If unsure, you can check what account you are using in the _Google Home_ app.
```

Comment:
```suggestion
The account you are using with the integration must have access to your google devices.
If unsure, you can check what account and account permissions in the mobile _Google Home_ app.
```

Reply:
@leikoilja Sorry, I think the second part became worse :)

How about
```markdown
If unsure, please check what account you are using in the _Google Home_ app and if you have these devices listed there.
```

Reply:
haha sound good, thanks for the spot

Reply:
Yup, this sounds the best

Reply:
Slightly change it, what do you guys think?